### PR TITLE
Add release note for RBAC functionality

### PR DESCRIPTION
**Pull request by matriv**
Closes: #12109

### CODE REVIEW

File: docs/appendices/release-notes/5.6.0.rst [hunk 1]
```diff
@@ -146,9 +147,16 @@ Performance and Resilience Improvements
       │  └ Collect[doc.b | [b] | (b > 1)] (rows=unknown)
       └ Collect[doc.c | [c] | true] (rows=unknown)
 
+
+.. _5.6.0_admin_ops:
```
Comment (seut) [inline docs/appendices/release-notes/5.6.0.rst:151]: Seems not needed? If so, I'd remove it..

**Review by seut — APPROVED** [review]

File: docs/appendices/release-notes/5.6.0.rst (hunk 1 above)
Comment (matriv) [reply docs/appendices/release-notes/5.6.0.rst:151]: I use it here: https://github.com/crate/crate/pull/15307/files#diff-682845acf34e36c6d05534a066bbd558f7f2b4a3273bee7e923aa68c9960dd76R82 but if you think it's too much I can remove.

Comment (seut) [reply docs/appendices/release-notes/5.6.0.rst:151]: Ah I see. You could also use `Administration and Operations`_ inside the same file. But no strong opinion.

Comment (matriv) [reply docs/appendices/release-notes/5.6.0.rst]: yep, changed it per your suggestion, thx!

**Pull request merged**